pvanalytics/system.py

```diff
@@ -25,11 +25,11 @@ def _peak_times(data):
     ) + peak_minutes
 
 
-def infer_orientation_least_squares(power_or_poa, daytime, tilts,
-                                    azimuths, solar_azimuth,
-                                    solar_zenith, ghi, dhi, dni):
-    """Determine system azimuth and tilt from power or POA using least
-    squares.
+def infer_orientation_solarnoon(power_or_poa, daytime, tilts,
+                                azimuths, solar_azimuth,
+                                solar_zenith, ghi, dhi, dni):
+    """Determine system azimuth and tilt from power or POA using apparent
```

Comment:
```suggestion
    """Determine system azimuth and tilt from power or POA using inferred
```

'apparent' has a specific meaning in solar position.